libcontainer/cgroups/fs2/rdma.go

```diff
@@ -0,0 +1,64 @@
+// +build linux
+
+package fs2
+
+import (
+	"strconv"
+
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/cgroups/fscommon"
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+func isRdmaSet(r *configs.Resources) bool {
+	return len(r.Rdma) > 0
+}
+
+func createCmdString(device string, limits configs.LinuxRdma) string {
+	cmdString := device
+	if limits.HcaHandles != nil {
+		cmdString += " hca_handle=" + strconv.FormatUint(uint64(*limits.HcaHandles), 10)
+	}
+	if limits.HcaObjects != nil {
+		cmdString += " hca_object=" + strconv.FormatUint(uint64(*limits.HcaObjects), 10)
+	}
+	return cmdString
+}
+
+func setRdma(path string, r *configs.Resources) error {
+	if !isRdmaSet(r) {
+		return nil
+	}
```

Comment:
There's no need to have this, as the following `for` loop will have 0 iterations when `len(r.Rdma) == 0` anyway.

Reply:
@kolyshkin Sure let me add relevant changes.

Reply:
@kolyshkin added this change in latest commit.